pkg/gofr/gofr_test.go

```diff
@@ -218,18 +219,24 @@ func Test_otelErrorHandler(t *testing.T) {
 }
 
 func Test_addRoute(t *testing.T) {
+	// Mock the command-line arguments to simulate running the "log" sub-command.
+	os.Args = []string{"", "log"}
```

Comment:
This sounds a bit dirty. You might face race issues at some points with tests

Reply:
I don't think that there is a problem with this since tests are not running in parallel, specially the ones changing the os.Args global variable. For a better practice I can recommend setting os.Args to their original state when tests are run with some command line arguments.

Reply:
I will try to fix that ! @vipul-rawat  @ccoVeille

Reply:
> The problem will occur when test are running in parallel and in the test that are changing the os.Args global variable. For a better practice I can recommend setting os.Args to their original state when tests are run with some command line arguments.

Yes, it's a common thing to do it with a t.Cleanup (cleaner than a defer)

I had to do it in some other go projects using cobra.

I'm a bit surprised there is no cleaner way to do it than overloading `os.Args` when using cobra.

Please note, I'm doing this commit from my phone. I didn't look at the code